### PR TITLE
the proper Union and DataType display

### PR DESCRIPTION
**Pull request by w00lf**
https://github.com/metanorma/ogc-citygml-xmi/issues/60

### CODE REVIEW

File: lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid [hunk 1]
```diff
@@ -72,18 +73,23 @@
...
-{% if package.data_types.size > 0 %}
-{% include "packages_entity_list_classes", classes: package.data_types, type: "Data types", depth: nested_depth %}
+{% assign non_unions = package.data_types | where: "stereotype", "DataType"  %}
+{% if non_unions.size > 0 %}
+{% include "packages_entity_list_classes", classes: non_unions, type: "Data types", depth: nested_depth %}
 {% endif %}
 
 {% if basic_types.size > 0 %}
-{% include "packages_entity_list_classes", classes: basic_types, type: "Primitive data types", depth: nested_depth %}
+{% include "packages_entity_list_classes", classes: basic_types, type: "Primitive Data Types" %}
```
Comment (ronaldtse) [inline lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid:82]: Need to return the original line.

File: lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid [hunk 2]
```diff
@@ -72,18 +73,23 @@
...
 {% endif %}
 
 {% if package.enums.size > 0 %}
 {% include "packages_entity_list_classes", classes: package.enums, type: "Enumerated classes", depth: nested_depth %}
 {% endif %}
 
+{% if unions.size > 0 %}
+{% include "packages_entity_list_classes", classes: unions, type: "Union types" %}
```
Comment (ronaldtse) [inline lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid:90]: Need to add `, depth: nested_depth`.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid (hunk 1 above)
Comment (w00lf) [reply lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid]: Done

File: lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid (hunk 2 above)
Comment (w00lf) [reply lib/metanorma/plugin/lutaml/liquid_templates/_packages_entity_list.liquid]: done

**Comment by ronaldtse** [comment]
Thank you @w00lf !

**Pull request merged**